app.py

```diff
@@ -1,33 +1,85 @@
-from flask import Flask
+from flask import Flask, g, json, jsonify, render_template, redirect, request, url_for
 from flask_cors import CORS
 from werkzeug import exceptions
 import sqlite3
+from controllers import urls
+
 
 app = Flask(__name__)
 CORS(app)
 
 DATABASE = 'database/database.db'
+URL = 'shoter.io'
 
-@app.route("/")
```

Comment:
Looks like at the moment controllers throw HTTP errors, maybe you could have error handlers in your routes? that way you could provide users a link to return to the home page and also catch other server errors? probably not needed but could be nice addition